tests/unit/components/ParameterForm.spec.ts

```diff
@@ -120,7 +129,7 @@ describe("parameter form", () => {
       },
     });
 
-    const component = await mountSuspended(ParameterForm, {
+    const component = mount(ParameterForm, {
```

Comment:
This change might not be necessary.

Reply:
Yep, confirmed we can switch this back:

```suggestion
    const component = await mountSuspended(ParameterForm, {
```

Reply:
I committed this suggestion